src/printer.rs

```diff
@@ -424,7 +424,9 @@ impl<'a> Printer for InteractivePrinter<'a> {
         }
 
         // Line contents.
-        if self.config.wrapping_mode == WrappingMode::NoWrapping {
+        if self.config.wrapping_mode == WrappingMode::NoWrapping(false)
+            || self.config.wrapping_mode == WrappingMode::NoWrapping(true)
+        {
```

Comment:
I think this could be:
```rust
if let WrappingMode::NoWrapping(_) = self.config.wrapping_mode {
  // …
```

Not really easier to read though. Another option would be to use the `matches!` macro. But we would need to bump the minimum supported Rust version for that. I'd be okay with this. We could use the `matches!` macro in other places as well.

Reply:
Yeah that looks a lot cleaner. Thanks!